docs/index.rst

```diff
@@ -1,22 +1 @@
-.. SymbiFlow V2X documentation master file
-
-Welcome to SymbiFlow Verilog to XML
-=========================================
-
-`SymbiFlow Verilog to XML`_ `V2X`_ .
-
-.. _SymbiFlow Verilog to XML: https://github.com/SymbiFlow/python-symbiflow-v2x
-.. _V2X: http://symbiflow-v2x.readthedocs.io/
-
```

Comment:
These toctree elements need to remain in the `docs/index.rst` otherwise those files won't end up in the documentation!

Reply:
Oops missed it